rules/stylistic-issues.js

```diff
@@ -0,0 +1,70 @@
+// Stylistic Issues http://eslint.org/docs/rules/#stylistic-issues
+
+module.exports = {
+	"rules": {
+		"array-bracket-spacing": [ 2, "always" ], // JSCS enforce spacing inside array brackets (fixable)
+		"block-spacing": [ 2, "always" ], // disallow or enforce spaces inside of single line blocks (fixable)
+		"brace-style": [ 2, "1tbs" ], // JSCS enforce one true brace style
+		"camelcase": [ 2, { "properties": "always" } ], // JSCS require camel case names
+		"comma-spacing": [ 2, { "before": false, "after": true } ], // enforce spacing before and after comma (fixable)
+		"comma-style": [ 2, "last" ], // JSCS enforce one true comma style
+		"computed-property-spacing": [ 2, "always" ], // require or disallow padding inside computed properties (fixable)
+		"consistent-this": [ 2, "self" ], // enforce consistent naming when capturing the current execution context
+		"eol-last": 2, // JSCS enforce newline at the end of file, with no multiple empty lines (fixable)
+		"func-names": 0, // require function expressions to have a name
+		"func-style": 0, // enforce use of function declarations or expressions
+		"id-length": [ 2, { "min": 1, "max": 100, "properties": "always", "exceptions": [] } ], // this option enforces minimum and maximum identifier lengths (variable names, property names etc.)
+		"id-match": 0, // require identifiers to match the provided regular expression
+		"indent": [ 2, "tab", { "SwitchCase": 1 } ], // JSCS specify tab or space width for your code (fixable)
+		"jsx-quotes": [ 2, "prefer-double" ], // specify whether double or single quotes should be used in JSX attributes
+		"key-spacing": [ 2, { "beforeColon": false, "afterColon": true } ], // JSCS enforce spacing between keys and values in object literal properties
+		"linebreak-style": [ 2, "unix" ], // JSCS disallow mixed 'LF' and 'CRLF' as linebreaks
+		"lines-around-comment": 0, // JSCS enforce empty lines around comments
+		"max-depth": [ 2, 5 ], // specify the maximum depth that blocks can be nested
+		"max-len": 0, // specify the maximum length of a line in your program
+		"max-nested-callbacks": [ 2, 3 ], // specify the maximum depth callbacks can be nested
```

Comment:
Personally I think this should be turned on, but I'm not sure what value we should set our default to be. **NOTE: I bumped up this value in the `test.js` set of rules because mocha nesting can get very deep when organizing test groups.**

http://eslint.org/docs/rules/max-nested-callbacks.html